content/docs/started/workstation/minikube-linux.md

```diff
@@ -327,3 +327,16 @@ Below is an example of accessing a Jupyter notebook hosted in a Linux machine re
    localhost:8889
    ```
    
+## TroubleShooting
+### AttributeError when create namespace object
+During execution of `Use Kubeflow fairing to build the docker image and launch a TFJob for training`, you may see the following error:
+```
+AttributeError: 'V1TFJob' object has no attribute 'openapi_types'
+```
+This is caused by an [existing issue in Kubernetes client API](https://github.com/kubernetes-client/python/issues/1112).
+
+To bypass this issue, please install kubernetes 10.0.1
```

Comment:
Is this the right version number? See https://github.com/kubernetes/kubernetes/releases

*Update:* Ah wait - this is referring to the Kubernetes client API. It'd be best to include that fact in the instruction, otherwise other people may be as confused as I was. :)

Reply:
Sure. Will do.